test/common/index.js

```diff
@@ -55,8 +54,16 @@ exports.isLinux = process.platform === 'linux';
 exports.isOSX = process.platform === 'darwin';
 
 exports.enoughTestMem = os.totalmem() > 0x40000000; /* 1 Gb */
-exports.bufferMaxSizeMsg = new RegExp(
-  `^RangeError: "size" argument must not be larger than ${buffer.kMaxLength}$`);
+exports.bufferMaxSizeMsg = expectsError({
+  code: 'ERR_INVALID_OPT_VALUE',
+  type: RangeError,
+  message: /^The value ".*" is invalid for option "size"$/
```

Comment:
Could you replace `.*` with `[^"]*` just to be more explicit
(and at L65 too)

Since `common` is becoming a [`Winnebago`](https://sourcemaking.com/antipatterns/the-blob), I'm actually not in love with these two. The old one is reused 3 times, and the new one 2 times.
Maybe for your next PR you could inline them 😉